cuda_next/include/detail/dimensions.hpp

```diff
@@ -0,0 +1,109 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Next in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _CUDA_NEXT_DETAIL_DIMENSIONS
+#define _CUDA_NEXT_DETAIL_DIMENSIONS
+
+#include <cuda/std/mdspan>
+
+namespace cuda_next
+{
+
+template <typename T, size_t... Extents>
+using dimensions = ::cuda::std::extents<T, Extents...>;
```

Comment:
Important:
* We should be clear whether we see this as a user driven testbed or whether we already apply strict coding rules from libcu++. This would e.g require reserved identifiers etc

Reply:
We decided that this is going to be a "user-land" library, so no __ugly names etc